vent/menus/main.py

```diff
@@ -134,6 +136,24 @@ def while_waiting(self):
         self.addfield6.display()
         self.addfield7.display()
 
+        # if file drop location changes deal with it
+        logger = Logger(__name__)
+        if self.file_drop.value != DropLocation():
```

Comment:
I'm afraid I don't quite understand what the question is. I'm not doing a check inside `create`.